app/services/get_application_forms_waiting_for_provider_decision.rb

```diff
@@ -0,0 +1,12 @@
+class GetApplicationFormsWaitingForProviderDecision
+  def self.call
+    ApplicationChoice
+      .where(status: :awaiting_provider_decision)
+      .where('reject_by_default_at < ?', chase_provider_time_limit)
+      .where.not(id: ChaserSent.provider_decision_request.select(:chased_id))
```

Comment:
This could get expensive as the database grows because `ChaserSent.provider_decision_request.map` is loading all the `ChaserSent` activerecord objects for that type (ever). I think you could use `pluck` to avoid instantiating the activerecord objects but I think a better way would be to do an outer join combined with an is null condition to get the DB to do all the heavy lifting.

You can do something like: `ApplicationChoice.left_outer_joins(:chasers_sent).where(chasers_sent: { id: nil })` if you set up the `ApplicationChoice#chasers_sent` association or use SQL.